seedmtk.mk

```diff
@@ -22,5 +22,6 @@ $(call inherit-product-if-exists, device/mediatek/sprout-common/sprout.mk)
 
 PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/rootdir/root/init.sprout.rc:root/init.sprout.rc \
-    $(LOCAL_PATH)/rootdir/root/fstab.sprout:root/fstab.sprout	\
-    $(LOCAL_PATH)/rootdir/root/kernel:kernel
+    $(LOCAL_PATH)/rootdir/root/fstab.sprout:root/fstab.sprout \
+    $(LOCAL_PATH)/rootdir/root/kernel:kernel \
+    $(LOCAL_PATH)/rootdir/recovery/twrp.fstab:root/etc/twrp.fstab
```

Comment:
By Referring on XDA here : [[DEV]How to compile TWRP touch recovery](https://forum.xda-developers.com/showthread.php?t=1943625).